hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestTimeTravelQuery.scala

```diff
@@ -238,6 +238,7 @@ class TestTimeTravelQuery extends HoodieClientTestBase {
       .options(commonOpts)
       .option(DataSourceWriteOptions.TABLE_TYPE.key, tableType.name())
       .option(PARTITIONPATH_FIELD.key, "")
+      .option(KEYGENERATOR_CLASS_NAME.key, classOf[NonpartitionedKeyGenerator].getName)
```

Comment:
I'd rather remove`.option(PARTITIONPATH_FIELD.key, "")` instead of setting keygen. That way the default behavior gets tested.